### ISSUE TITLE
repo-install test is failing on daily-iso: *** /etc/yum.repos.d/test03.repo should specify the cost option

### ISSUE THREAD

**Issue by rvykydal** [rhinstaller/kickstart-tests#662]
repo-install test is failing with:
```
2022-02-10 23:55:23,405 DEBUG: Failed to run subprocess: '['/opt/kstest/kickstart-tests/scripts/launcher/lib/launcher_interface.sh', '-i', '/opt/kstest/data/images/boot.iso', '-k', '1', '-w', '/var/tmp/kstest-repo-install.2022_02_10-23_41_44.uob8_2sw', '-t', '/opt/kstest/kickstart-tests/repo-install.sh', 'validate']'
stdout:
*** /etc/yum.repos.d/test03.repo should specify the cost option
[test03]
name = test03
enabled = 1
baseurl = http://10.0.2.2:40004//repo
cost = 25
*** /etc/yum.repos.d/test04.repo should specify the sslverify option
[test04]
name = test04
enabled = 1
baseurl = http://10.0.2.2:40004//repo
sslverify = 0
*** /etc/yum.repos.d/test05.repo should specify the includepkgs option
[test05]
name = test05
enabled = 1
baseurl = http://10.0.2.2:40004//repo
includepkgs = p1, p2
excludepkgs = p3, p4

2022-02-10 23:55:23,405 INFO: RESULT:repo-install:d7fe692d2b33:FAILED:Validation failed with return code 1
```
[kstest.log](https://github.com/rhinstaller/kickstart-tests/files/8047064/kstest.log)
[virt-install.log](https://github.com/rhinstaller/kickstart-tests/files/8047065/virt-install.log)

**Comment by rvykydal** [comment]
PR to disable the test: https://github.com/rhinstaller/kickstart-tests/pull/663

**Comment by rvykydal** [comment]
Will be fixed by https://github.com/rhinstaller/kickstart-tests/pull/653, I'll check and close this one.

**Comment by rvykydal** [comment]
> Will be fixed by #653, I'll check and close this one.

Confirmed
